app/assets/javascripts/course/assessment/submission/submission.js

```diff
@@ -164,8 +164,16 @@
       $('#' + identifier).find('textarea.code').ace();
     });
 
-    // Show the first tab on page load.
-    $(MULTI_QUESTION_ASSESSMENT_SELECTOR + '.tab-header a:first').tab('show');
+    // Check the URL for a step parameter
+    var url = document.location.toString();
+    if (url.match('step=')) {
```

Comment:
I decided not to abstract out `renderTab` as building the selector is too nasty. In this case, it's actually one of the HTML attributes, but other cases might use another attribute.

Perhaps template strings would help.